.travis.yml

```diff
@@ -1,67 +1,34 @@
-# We set the language to c because python isn't supported on the MacOS X nodes
-# on Travis. However, the language ends up being irrelevant anyway, since we
-# install Python ourselves using conda.
+# Most of the CI testing happens on Azure, but this configuration is for a
+# regular cron job on Travis (as well as for pull requests).
```

Comment:
I don't think we need to keep this for RPs, the azure matrix is comprehensive enough.

Reply:
Ok, I'll set EVENT_TYPE=cron

Reply:
actually it's better to be handled by the travis api directly rather than going though the ci-helpers hack. Check for `TRAVIS_EVENT_TYPE` being cron, then it won't be kick off.

Reply:
maybe you can also say `if: type = cron`, but I'm not sure that would work outside of the stages setup